unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs

```diff
@@ -69,6 +73,7 @@ private void Awake()
             }
 
             cameraChangeAction.OnTriggered += OnCameraChangeAction;
+            mouseWheelAction.OnValueChanged += OnMouseWheelChangeValue;
```

Comment:
We should unsubscribe this on the `OnDestroy` method.

Reply:
Done!